std_msgs/Int16[] top_left_x
std_msgs/Int16[] top_left_y
std_msgs/Int16[] bottom_right_x
std_msgs/Int16[] bottom_right_y
std_msgs/Float32[] distance


